esp/Bridge/src/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <Arduino.h>
```

Comment:
I usually ask to have a short description of the module at the beginning of the file, but if I understand the code correctly then it reads Serial and writes to Serial1 a vice versa.

Reply:
p.s. I see the code is already copied on Robotika website :)
https://robotika.cz/articles/digitalservo/cs